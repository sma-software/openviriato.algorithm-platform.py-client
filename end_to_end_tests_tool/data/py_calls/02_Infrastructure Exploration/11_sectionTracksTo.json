{
    "PythonParameterMapping": {
      "to_node_id" : "$.queryParameters.[0].value"
    }
}
