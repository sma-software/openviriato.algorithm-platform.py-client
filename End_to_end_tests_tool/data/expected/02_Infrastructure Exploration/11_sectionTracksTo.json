[
  {
    "id": 883,
    "code": "955",
    "sectionCode": "61005",
    "weight": 22260,
    "debugString": "sectiontrack:s_61005 n_85NH 955"
  },
  {
    "id": 884,
    "code": "855",
    "sectionCode": "61005",
    "weight": 22260,
    "debugString": "sectiontrack:s_61005 n_85NH 855"
  },
  {
    "id": 1157,
    "code": "752",
    "sectionCode": "73000",
    "weight": 13440,
    "debugString": "sectiontrack:s_73000 n_85NH 752"
  },
  {
    "id": 885,
    "code": "838",
    "sectionCode": "61010",
    "weight": 37040,
    "debugString": "sectiontrack:s_61010 n_85AR 838"
  },
  {
    "id": 886,
    "code": "2",
    "sectionCode": "61010",
    "weight": 37040,
    "debugString": "sectiontrack:s_61010 n_85AR 2"
  }
]